docs/website/docs/building-from-source/getting-started.md

```diff
@@ -138,32 +144,102 @@ cmake --build ../iree-build/
 
     See also our [developer documentation for ccache](https://github.com/openxla/iree/blob/main/docs/developers/developing_iree/ccache.md).
 
-## What's next?
+### Optional components
+
+By default, the CMake build includes:
+
+* All compiler targets (`llvm-cpu`, `cuda`, `vulkan-spirv`, etc.)
+* All runtime HAL drivers (`local-task`, `cuda`, `vulkan`, etc.)
+* All compiler input formats (StableHLO, TOSA, etc.)
+* All compiler output formats (VM bytecode, C)
 
-<!-- TODO(scotttodd): "at this point you can..." -->
+The default build does _not_ include:
+
+* Compiler or runtime bindings (Python, TFLite, etc.)
+* Advanced features like AddressSanitizer or tracing instrumentation
+* Experimental components
+
+These can be changed via the `IREE_` CMake options listed in the root
+[`CMakeLists.txt`](https://github.com/openxla/iree/blob/main/CMakeLists.txt).
+
+### Extensions and project integrations
+
+When using IREE within other projects, you can register compiler plugins and
+runtime HAL drivers. You can also bring your own copy of LLVM and some other
+tools. See the root
+[`CMakeLists.txt`](https://github.com/openxla/iree/blob/main/CMakeLists.txt)
+for details.
+
+## Tests and samples
 
 ### Running tests
 
-Build test dependencies and run tests:
+Tests are run via [ctest](https://cmake.org/cmake/help/latest/manual/ctest.1.html).
+To build and run the core project tests:
 
 ``` shell
-cmake --build ../iree-build --target iree-run-tests
+# Build default targets
+cmake --build ../iree-build/
+
+# Run tests
+ctest --test-dir ../iree-build/
 ```
 
-Internally, this builds dependencies via the `iree-test-deps` target and
-invokes [CTest](https://gitlab.kitware.com/cmake/community/-/wikis/doc/ctest/Testing-With-CTest).
+!!! Caution
+    This has two limitations:
+
+    1. Large tests are excluded from the build by default
+    2. Some tests require hardware like a GPU and will fail on unsupported systems
```

Comment:
If I run `ctest --test-dir ../iree-build/` on my Mac or gLinux, will it just work? It would be nice if we had a test option that would exclude tests that aren't supported by the hardware (I realize that might not be feasible).

Reply:
Never mind. I just saw the section about the helper script that uses env vars to create filters.

Reply:
> It would be nice if we had a test option that would exclude tests that aren't supported by the hardware (I realize that might not be feasible).

Discussion with some ideas at https://github.com/openxla/iree/issues/12042